### PR TITLE
[fix](regression) fix export file case

### PR DESCRIPTION
**Pull request by zgxme**
## Proposed changes

fix 2 regression problems
1.The machine that performs the export is randomized when there are multiple nodes.
2.`s3ExportBucketName` not in the daily regression conf.

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zgxme** [comment]
run buildall

**Comment by zgxme** [comment]
run buildall

File: regression-test/suites/export/test_array_export.groovy [hunk 1]
```diff
@@ -165,9 +165,18 @@ suite("test_array_export", "export") {
         } else {
             throw new IllegalStateException("""${outFilePath} already exists! """)
         }
-        sql """
-            SELECT * FROM ${tableName} t ORDER BY k1 INTO OUTFILE "file://${outFilePath}/";
+        result = sql """
+            SELECT * FROM ${tableName} t ORDER BY k1 INTO OUTFILE "file://${outFile}/";
         """
+        url = result[0][3]
+        urlHost = url.substring(8, url.indexOf("${outFile}"))
```
Comment (shuke987) [inline regression-test/suites/export/test_array_export.groovy:172]: can we pack scp and delete into a function( like plugin ) and call it

Comment (zgxme) [reply regression-test/suites/export/test_array_export.groovy:173]: done

**Comment by zgxme** [comment]
run buildall

**Review by shuke987 — APPROVED** [review]

**Review by shuke987 — APPROVED** [review]

**Review by shuke987 — APPROVED** [review]

**Review by shuke987 — APPROVED** [review]

**Review by xiaokang — APPROVED** [review]
LGTM

**Comment by xiaokang** [comment]
@zgxme add some description please

**Comment by zgxme** [comment]
> @zgxme add some description please

done

**Pull request merged**